app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -348,6 +362,33 @@ export default class SampleViewV2 extends React.Component {
     return parsedValue;
   };
 
+  adjustMetricPrecision = species => {
+    Object.entries(species).forEach(([countType, metricValue]) => {
```

Comment:
Ok, I might have induced you in error. Here we don't yet know if it is a countType or a metricKey... so, lets call it key only.

Reply:
I am not sure if I understand the comments about `adjustMetricPrecision`. Here we talk about renaming `countType` to `key` - but then refer to replacing the two `else if` statements with one `if (countType === "nt" || countType === "nr")` in the other comment. Did you mean `if (key === "nt" || key === "nr")`?

Reply:
Yes :)